src/extractors/custom/techlog.iij.ad.jp/index.js

```diff
@@ -0,0 +1,31 @@
+export const TechlogIijAdJpExtractor = {
+  domain: 'techlog.iij.ad.jp',
+
+  title: {
+    selectors: ['h1.entry-title'],
+  },
+
+  author: {
+    selectors: ['a[rel="author"]'],
+  },
+
+  date_published: {
+    selectors: [['time', 'datetime']],
```

Comment:
Let's make this a bit more specific, so that it follows a similar selector structure as the other fields, so maybe `time.entry-date`?

Reply:
Thank you.
I fixed.